src/WidgetRegistry.ts

```diff
@@ -53,6 +53,13 @@ export function isWidgetBaseConstructor(item: any): item is WidgetBaseConstructo
 	return Boolean(item && item._type === WIDGET_BASE_TYPE);
 }
 
+function getEventType(label: RegistryLabel): RegistryLabel {
```

Comment:
I understood the code, but it was more of a broader question of if there is `type: 'loaded:foo'` for a string but `type: @@foo` for a Symbol that it seemed like a bigger challenge.  I know it is a broader question than this specific code, but it seems like symbols don't quite fit neatly into the `WidgetRegistry` event model.